assets/css/layouts/home.css

```diff
@@ -95,7 +95,7 @@
 	transition: all 0.2s;
 }
 .playground-header-figure-wrapper img {
-	transition: all 0.3s;
+	transition: all 0.1s;
```

Comment:
Testing this some more, I think I prefer 200 ms for the transition. It feels a bit more calm and also leaves us more time to load the HTML and main image.